AutoGraph/WebSocketClient.swift

```diff
@@ -298,6 +325,41 @@ extension WebSocketClient: WebSocketDelegate {
         }
     }
     
+    func processWebSocketTextEvent(text: String) throws {
+        guard let data = text.data(using: .utf8) else {
+            throw ResponseSerializerError.failedToConvertTextToData(text)
+        }
+        
+        let json = try JSONValue.decode(data)
+        guard case let .string(typeValue) = json["type"] else {
```

Comment:
would we ever expect to not have a `type`? should we throw if there is no `type`?

Reply:
There should always be `type`

Reply:
that seems like an error then, maybe worth throwing `webSocketEventMissingType` or something